[0.248s] Invoking command in '/home/plague/ros2_ws/src/examples/launch_testing/launch_testing_examples': AMENT_PREFIX_PATH=/home/plague/ros2_ws/install/launch_testing_examples:${AMENT_PREFIX_PATH} PYTEST_ADDOPTS=--tb=short --junit-xml=/home/plague/ros2_ws/build/launch_testing_examples/pytest.xml --junit-prefix=launch_testing_examples -o cache_dir=/home/plague/ros2_ws/build/launch_testing_examples/.pytest_cache PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=/home/plague/ros2_ws/build/launch_testing_examples:/home/plague/ros2_ws/install/launch_testing_examples/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 -m pytest
[0.488s] [1m============================= test session starts ==============================[0m
[0.488s] platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
[0.492s] cachedir: /home/plague/ros2_ws/build/launch_testing_examples/.pytest_cache
[0.492s] rootdir: /home/plague/ros2_ws/src/examples/launch_testing/launch_testing_examples
[0.492s] plugins: ament-flake8-0.14.3, ament-xmllint-0.14.3, ament-lint-0.14.3, launch-testing-2.0.2, ament-copyright-0.14.3, launch-testing-ros-0.24.1, ament-pep257-0.14.3, cov-3.0.0, colcon-core-0.16.0
[0.746s] [1mcollecting ... [0m[1m
[0.747s] collected 9 items                                                              [0m
[0.747s] 
[1.934s] launch_testing_examples/check_msgs_launch_test.py [32m.[0m[33m                      [ 11%][0m
[23.746s] launch_testing_examples/check_multiple_nodes_launch_test.py [32m.[0m[33m            [ 22%][0m
[29.087s] launch_testing_examples/check_node_launch_test.py [32m.[0m[33m                      [ 33%][0m
[29.092s] launch_testing_examples/hello_world_launch_test.py [32m.[0m[33m                     [ 44%][0m
[32.283s] launch_testing_examples/record_rosbag_launch_test.py [32m.[0m[33m                   [ 55%][0m
[32.659s] launch_testing_examples/set_param_launch_test.py [32m.[0m[33m                       [ 66%][0m
[32.883s] test/test_copyright.py [32m.[0m[33m                                                 [ 77%][0m
[32.988s] test/test_flake8.py [32m.[0m[33m                                                    [ 88%][0m
[33.019s] test/test_pep257.py [32m.[0m[33m                                                    [100%][0m
[33.019s] 
[33.019s] [33m=============================== warnings summary ===============================[0m
[33.019s] launch_testing_examples/check_msgs_launch_test.py::launch_testing_examples.check_msgs_launch_test
[33.019s]   Warning: Exception in thread Thread-1 (spin)
[33.019s]   
[33.019s]   Traceback (most recent call last):
[33.019s]     File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
[33.019s]       self.run()
[33.019s]     File "/usr/lib/python3.10/threading.py", line 953, in run
[33.019s]       self._target(*self._args, **self._kwargs)
[33.019s]     File "/opt/ros/iron/lib/python3.10/site-packages/rclpy/executors.py", line 295, in spin
[33.019s]       self.spin_once()
[33.019s]     File "/opt/ros/iron/lib/python3.10/site-packages/rclpy/executors.py", line 748, in spin_once
[33.019s]       self._spin_once_impl(timeout_sec)
[33.019s]     File "/opt/ros/iron/lib/python3.10/site-packages/rclpy/executors.py", line 737, in _spin_once_impl
[33.019s]       handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[33.019s]     File "/opt/ros/iron/lib/python3.10/site-packages/rclpy/executors.py", line 723, in wait_for_ready_callbacks
[33.019s]       return next(self._cb_iter)
[33.019s]     File "/opt/ros/iron/lib/python3.10/site-packages/rclpy/executors.py", line 624, in _wait_for_ready_callbacks
[33.019s]       raise ExternalShutdownException()
[33.019s]   rclpy.executors.ExternalShutdownException
[33.020s] 
[33.020s] test/test_flake8.py::test_flake8
[33.020s] test/test_flake8.py::test_flake8
[33.020s]   Warning: SelectableGroups dict interface is deprecated. Use select.
[33.020s] 
[33.020s] -- Docs: https://docs.pytest.org/en/stable/warnings.html
[33.020s] - generated xml file: /home/plague/ros2_ws/build/launch_testing_examples/pytest.xml -
[33.020s] [33m======================== [32m9 passed[0m, [33m[1m3 warnings[0m[33m in 32.53s[0m[33m ========================[0m
[33.249s] Invoked command in '/home/plague/ros2_ws/src/examples/launch_testing/launch_testing_examples' returned '0': AMENT_PREFIX_PATH=/home/plague/ros2_ws/install/launch_testing_examples:${AMENT_PREFIX_PATH} PYTEST_ADDOPTS=--tb=short --junit-xml=/home/plague/ros2_ws/build/launch_testing_examples/pytest.xml --junit-prefix=launch_testing_examples -o cache_dir=/home/plague/ros2_ws/build/launch_testing_examples/.pytest_cache PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=/home/plague/ros2_ws/build/launch_testing_examples:/home/plague/ros2_ws/install/launch_testing_examples/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 -m pytest
